apps/wallet/src/ui/app/pages/home/interstitial/index.tsx

```diff
@@ -76,6 +76,8 @@ function Interstitial({
 					className="flex appearance-none border-none rounded-full bg-[#4CA2FF] h-10 px-6 cursor-pointer items-center no-underline"
 					onClick={onClick}
 					to={bannerUrl}
+					target="_blank"
```

Comment:
Target `_blank` on an anchor element  implies `noopener` for browser versions released 3 years or less ago. see here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility but should be fine to keep it I guess for older browsers.